README.md

```diff
@@ -1,12 +1,103 @@
+- **Documentation and Tutorials:** Contribute to comprehensive documentation and tutorials that guide users through the platform effectively.
 
 ## Demo
 
-This code is deployed at: [https://playground.accordproject.org](https://playground.accordproject.org)
\ No newline at end of file
+The Template Playground is deployed at: [https://playground.accordproject.org](https://playground.accordproject.org)
+
+---
+
+<p align="center">
+  <a href="https://www.accordproject.org/">
+    <img src="public/APLogo.png" alt="Accord Project Logo" width="400" />
+  </a>
+</p>
+
+<p align="center">
+  <a href="./LICENSE">
+    <img src="https://img.shields.io/github/license/accordproject/cicero?color=bright-green" alt="GitHub license">
+  </a>
+  <a href="https://discord.gg/Zm99SKhhtA">
+    <img src="https://img.shields.io/badge/Accord%20Project-Join%20Discord-blue" alt="Join the Accord Project Discord"/>
+  </a>
+</p>
+
+Accord Project is an open source, non-profit, initiative working to transform contract management and contract automation by digitizing contracts. Accord Project operates under the umbrella of the [Linux Foundation][linuxfound]. The technical charter for the Accord Project can be found [here][charter].
+
+## Learn More About Accord Project
+
+### [Overview][apmain]
+
+### [Documentation][apdoc]
+
+## Contributing
+
+The Accord Project technology is being developed as open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
+
+Find out what’s coming on our [blog][apblog].
+
+Join the Accord Project Technology Working Group [Discord channel][apslack] to get involved!
+
+For code contributions, read our [CONTRIBUTING guide][contributing] and information for [DEVELOPERS][developers].
+
+### README Badge
+
+Using Accord Project? Add a README badge to let everyone know: [![accord project](https://img.shields.io/badge/powered%20by-accord%20project-19C6C8.svg)](https://www.accordproject.org/)
+
+```
+[![accord project](https://img.shields.io/badge/powered%20by-accord%20project-19C6C8.svg)](https://www.accordproject.org/)
+```
+
+## License <a name="license"></a>
+
+Accord Project source code files are made available under the [Apache License, Version 2.0][apache].
+Accord Project documentation files are made available under the [Creative Commons Attribution 4.0 International License][creativecommons] (CC-BY-4.0).
+
+Copyright 2018-2019 Clause, Inc. All trademarks are the property of their respective owners. See [LF Projects Trademark Policy](https://lfprojects.org/policies/trademark-policy/).
+
+[linuxfound]: https://www.linuxfoundation.org
+[charter]: https://github.com/accordproject/governance/blob/master/accord-project-technical-charter.md
+[apmain]: https://accordproject.org/
+[apblog]: https://medium.com/@accordhq
```

Comment:
Can you update this to point to https://accordproject.org/news ?

Reply:
Alright! Sure @DianaLease On it

Reply:
✅ Updated now @DianaLease, Good to go